src/App.Metrics.Formatters.Prometheus/MetricsPrometheusOptions.cs

```diff
@@ -18,5 +18,7 @@ public MetricsPrometheusOptions()
         }
 
         public Func<string, string, string> MetricNameFormatter { get; set; }
+
+        public NewLineFormat NewLineFormat { get; set; } = NewLineFormat.Auto;
```

Comment:
For formatting for Prometheus shouldn't this always be Unix regardless of platform the metrics are running on?